### ISSUE TITLE
Show-Modal don't show up on dynamic page

### ISSUE THREAD

**Issue by psott** [ironmansoftware/universal-dashboard#348]
Hi,
Show-Modal don't show up if placed on a dynamic page.

Test code:

```
Get-UDDashboard -Name 'test' | Stop-UDDashboard
$pages = @()
$pages += New-UDPage -Name 'Home' -Content {
  New-UDCard -Title "Show-Modal Static Page"
  New-UDButton -Text "This modal works" -OnClick {
        Show-UDModal -Header { New-UDHeading -Size 5 -Text "Show-Modal Static Page" } -Content {
            New-UDRow -Columns {
                New-UDColumn -Size 2 -Content {
                    New-UDButton -Text "Ok" -OnClick {
                        Hide-UDModal
                    }
                }
                New-UDColumn -Size 2 -Content {
                    New-UDButton -Text "Cancel" -OnClick {
                        Hide-UDModal
                    }
                }
            }
        } 
    }
    New-UDRow{
        New-UDButton -Text 'Link to dynamic page' -OnClick {Invoke-UDRedirect -Url "/object/srv1"}
    }
}

$pages += New-UDPage -Url "/object/:identity" -Endpoint {
  param($identity)
  New-UDCard -Title "Show-Modal dynamic Page: $identity"
  New-UDButton -Text "this modal don't work" -OnClick {
        Show-UDModal -Header { New-UDHeading -Size 5 -Text "Show-Modal dynamic Page: $identity" } -Content {
            New-UDRow -Columns {
                New-UDColumn -Size 2 -Content {
                    New-UDButton -Text "Ok" -OnClick {
                        Hide-UDModal
                    }
                }
                New-UDColumn -Size 2 -Content {
                    New-UDButton -Text "Cancel" -OnClick {
                        Hide-UDModal
                    }
                }
            }
        } 
    }
}

$dash = New-UDDashboard -Title 'test' -Pages $pages
Start-UDDashboard -Port 10001 -Dashboard $dash
```

Tested with 1.7.0, 2.0.0 and 2.0.1
PS 5.1 and PSCore 6.1.0rc
.NET Core 2.1
.NET Framework 4.7.2
Chrome 69 and Firefox 52

**Comment by psott** [comment]
tested this with the appveyor 2.1.0 build. looks like that version will fix it

**Comment by psott** [comment]
fixed with final 2.1.0 build, thx
